config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
```

Comment:
Should this include `password`?

Reply:
> Should this include `password`?

`:passw` already includes that, I believe it is a "match" instead of "eq", if you know what I mean.

I did a test, this is what I got from the logs:

```
Started GET "/?password=[FILTERED]" for ::1 at 2021-07-15 18:49:37 +1200
Processing by ApplicationController#home as */*
  Parameters: {"password"=>"[FILTERED]"}
  Rendering layout layouts/application.html.erb
  Rendering application/home.en.html.erb within layouts/application
  Rendered application/home.en.html.erb within layouts/application (Duration: 0.3ms | Allocations: 46)
[Webpacker] Everything's up-to-date. Nothing to do
  Rendered layout layouts/application.html.erb (Duration: 9.9ms | Allocations: 3042)
Completed 200 OK in 11ms (Views: 10.8ms | ActiveRecord: 0.0ms | Allocations: 3285)
```

Reply:
Ah, gotcha! I didn't know that. Neat.